packages/ui/src/components/va-card/VaCard.vue

```diff
@@ -22,8 +22,16 @@
 <script lang="ts">
 import { defineComponent, computed } from 'vue'
 
+import pick from 'lodash/pick'
```

Comment:
```suggestion
import pick from 'lodash/pick.js'
```
Cause of build issues.